### ISSUE TITLE
Create entrypoint script for docker to run migrate script

### ISSUE THREAD

**Issue by mattwr18** [Human-Connection/Human-Connection#220]
**Is your feature request related to a problem? Please describe.**
currently, one must remember to run the script as it creates unique constraints in the database

**Describe the solution you'd like**
create a entrypoint script that waits on the server and then runs the migrate script, and maybe seeds the database
**Describe alternatives you've considered**
create custom error to remind a user to run the script like rails does when one has pending migrations, this seems like it might be more work

**Comment by roschaefer** [comment]
Very related to #206

**Comment by mattwr18** [comment]
closing in favor of #206
